src/test/java/com/hocs/test/pages/decs/TimelineTab.java

```diff
@@ -221,4 +221,32 @@ public void assertCaseLogWithTitleContainsText(String caseLogTitle, String text)
             Assert.fail("The '" + caseLogTitle + "' case log was expected to have text: '" + caseLogContents + "' but had text: '" + text + "'");
         }
     }
+
+    public void AssertCaseNotesAuthoredByUserAreNotVisible(User user) {
```

Comment:
The A needs to be lowercase at the start of this method

Reply:
fixed